docs/mine-hnt/validators/mainnet/wallet.mdx

```diff
@@ -145,21 +145,22 @@ unstaked.
 ### Unstake Tokens
 
 If you unstake tokens the validator node enters a stake cooldown period. During this cooldown period
-the validator node will not earn rewards and at the end of the cooldown period (currently 250,000
-blocks or approximately 5.7 months in mainnet) your initial staking amount will be released to the
+the validator node will not earn rewards and at the end of the cooldown period (currently 125,000
+blocks or approximately 3 months in mainnet) your initial staking amount will be released to the
```

Comment:
```suggestion
blocks or approximately 86 days in mainnet) your initial staking amount will be released to the
```

Reply:
Anthonyra also suggested increasing the + 10 to + 30